src/toffy/normalize.py

```diff
@@ -707,7 +707,7 @@ def create_fitted_pulse_heights_file(
 
         # for non-autogain runs, use the default min_obs at 10 for curve fitting
         # for autogain runs, the median of all observed MPH values should always be used
-        min_obs = 0 if autogain else 10
+        min_obs = 10**10 if autogain else 10
```

Comment:
why did you switch from 0 to 10**10?

Reply:
I got the thresholding wrong. `min_obs` actually defines the minimum number of observations needed to run the curve-fitting process. However, this process should _never_ run for `autogain`. So the hack is to set it to an unreasonably high number that no dataset will ever reach in practice. In this way, the fallback median method is always used.

Reply:
Sounds good, thanks!